ambry-cloud/src/test/java/com/github/ambry/cloud/CloudBlobStoreTest.java

```diff
@@ -873,11 +940,11 @@ public void testPutWithTtl() throws Exception {
    * @return the generated {@link BlobId}.
    */
   private BlobId addBlobToSet(MockMessageWriteSet messageWriteSet, long size, long expiresAtMs, short accountId,
```

Comment:
Shouldn't these methods be removed here?

Reply:
done.